### DISCUSSION TITLE
System tray ChatGPT - Send message

### DISCUSSION THREAD

**Discussion by andrradar** [lencx/Noi#220] [General]
The system tray widget is nearly perfect in terms of both functionality and aesthetics, making it incredibly easy to access and use.

However, I’ve encountered an issue that significantly affects the user experience. Unfortunately, the ability to send messages using the keyboard (specifically, pressing Enter) is currently not working as expected. This issue persists even when trying other combinations like Ctrl+Enter, Shift+Enter, or Alt+Enter.

Having to click the send button manually somewhat diminishes the streamlined experience that your tool otherwise provides. Could you please prioritize fixing this bug so that users can send messages directly from the keyboard? This would greatly enhance the usability of the tool.

Thank you for your attention to this matter, and I look forward to the update.

**Comment by kerzhaev** [comment]
This bug came from the summer update of ChatGPT. Before that, it worked well.

**Reply by andrradar** [reply]
If I roll back the version - will it work? Or the problem is that ChatGPT has been updated and I need to update Noi?

**Reply by lencx** [reply]
@andrradar https://github.com/lencx/Noi/issues/172#issuecomment-2199585348
